source/jormungandr/jormungandr/instance.py

```diff
@@ -58,14 +63,14 @@
 from collections import deque
 
 type_to_pttype = {
-    "stop_area": request_pb2.PlaceCodeRequest.StopArea,
-    "network": request_pb2.PlaceCodeRequest.Network,
-    "company": request_pb2.PlaceCodeRequest.Company,
-    "line": request_pb2.PlaceCodeRequest.Line,
-    "route": request_pb2.PlaceCodeRequest.Route,
-    "vehicle_journey": request_pb2.PlaceCodeRequest.VehicleJourney,
-    "stop_point": request_pb2.PlaceCodeRequest.StopPoint,
-    "calendar": request_pb2.PlaceCodeRequest.Calendar,
+    "stop_area": request_pb2.PlaceCodeRequest.StopArea,  # type: ignore
```

Comment:
Here type is ignored because mypy would want a consistent type?
Should we use a parent of them all (well, probably object in that case)?

Reply:
There is no stub for protobuf so mypy as some problems with it.
I need to look at https://github.com/dropbox/mypy-protobuf.
The main objective of this PR was to introduce mypy and make test pass